bindings/python/iota_sdk/types/send_params.py

```diff
@@ -71,18 +71,15 @@ class CreateNativeTokenParams():
         foundry_metadata: The foundry metadata of the native token.
         account_id: The ID of the corresponding account.
     """
-    circulating_supply: int
-    maximum_supply: int
+    circulating_supply: int = field(metadata=config(
+        encoder=str
+    ))
+    maximum_supply: int = field(metadata=config(
+        encoder=str
```

Comment:
Are these going to be hex strings ?

Reply:
No, I opened an issue for it https://github.com/iotaledger/iota-sdk/issues/1494

Reply:
Hmm ok, I would have kept a custom dict because we are (temporarily) making this wrong. But if we won't forget to change it with the issue that's fine.